pkgs/tools/misc/flitter/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, ocamlPackages
+, fetchFromGitHub
+, makeWrapper
+, python3
+}:
+
+ocamlPackages.buildDunePackage rec {
+  pname = "flitter";
+  version = "unstable-2020-10-05";
```

Comment:
Please add a link to https://github.com/alexozer/flitter/issues/34 for future reference.